deploy/crds/core.humio.com_humioclusters_crd.yaml

```diff
@@ -2,12 +2,6 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: humioclusters.core.humio.com
-  labels:
-    app: 'humio-operator'
-    app.kubernetes.io/name: 'humio-operator'
-    app.kubernetes.io/instance: 'humio-operator'
-    app.kubernetes.io/managed-by: 'Helm'
-    helm.sh/chart: 'humio-operator-0.0.6'
```

Comment:
Hmm. Those labels should get added automatically when running "make crds". Did you generate the CRD's using that make target or did you run `operator-sdk generate crds` directly?

Reply:
fixed